code/game/machinery/computer/card.dm

```diff
@@ -37,7 +37,6 @@ GLOBAL_VAR_INIT(time_last_changed_position, 0)
 		"Research Director",
 		"Chief Medical Officer",
 		"Brig Physician",
-		"SolGov Representative",		//WS Edit - SolGov Rep
```

Comment:
This is gonna make solgov representative slots openable/closable by HoPs

Reply:
No, i added it back in a modularized file 😎